tests/unittests/interface_types.cpp

```diff
@@ -2,9 +2,7 @@
 
 #include "podio/ObjectID.h"
 
-#include "datamodel/AnotherTypeWithEnergy.h"
-#include "datamodel/EnergyInNamespaceCollection.h"
-#include "datamodel/ExampleClusterCollection.h"
+#include "datamodel/MutableExampleCluster.h"
 #include "datamodel/ExampleHitCollection.h"
```

Comment:
```suggestion
#include "datamodel/ExampleHitCollection.h"
#include "datamodel/MutableExampleCluster.h"
```
clang-format wants them ordered alphabetically